databases/week3/DB3.txt

```diff
@@ -0,0 +1,113 @@
+
+const database = 'DB_mongo';
+
+use(database);
+
+db.createCollection(collection);
+
+// The prototype form to create a regular collection:
+db.createCollection( <Meal>,
+  {
+    id: <number>,
+    title: <string>,
+    description_meal : <string>,
+    location_meal : <string>,
+    when_created: <string>,
+    max_reservations: <number>,
+    price: <number>,
+    created_date: <string>
+      }
+) 
+ 
+db.createCollection( <Reservation>,
+  {
+    id: <number>,
+    number_of_guests: <number>,
+    meal_id: <number>,
+    created_date:  <string>,
+    contact_phonenumber : <string>,
+    contact_name: <string>,
+    contact_email: <string>
+    
+      }
+) 
+
+db.createCollection( <Review>,
+  {
+    id: <number>,
+    title: <string>,
+    description_reviev: <string>,
+    meal_id: <number>,
+    stars: <number>,
+    created_date:  <string>,
+    contact_phonenumber : <string>,
+    contact_name: <string>,
+    contact_email: <string>
+    
+      }
+) 
```

Comment:
I'm not sure you can do this even if you remove `<>`s which are used to define placeholders in the documentation. (lines 9-48). If you wrote this for yourself then it should've been added as a comment. Otherwise, this is invalid syntax.

Reply:
done